instrumentation/rack/lib/opentelemetry/instrumentation/rack/version.rb

```diff
@@ -7,7 +7,7 @@
 module OpenTelemetry
   module Instrumentation
     module Rack
-      VERSION = '0.21.0'
+      VERSION = '0.21.1'
```

Comment:
```suggestion
      VERSION = '0.22.0'
```

Reply:
With [the changelog dialed down to only a FIXED](https://github.com/open-telemetry/opentelemetry-ruby-contrib/pull/134/files#r987253446), not ADDED & FIXED, the patch-bump is appropriate and the minor is not.